src/Data/Array.purs

```diff
@@ -324,6 +429,12 @@ foreign import findIndexImpl
   -> Maybe Int
 
 -- | Find the last index for which a predicate holds.
+-- |
+-- | ```purescript
+-- | findLastIndex (_ > 0) [-10, -4, 2, 3] = Just 3
```

Comment:
I think it might be slightly better to use a type other than `Int` for the elements of the array here, actually, just to avoid potential confusion from people thinking `findLastIndex` might return an element of the array vs an index into it.